src/tellor_disputables/cli.py

```diff
@@ -11,35 +11,38 @@
 
 from tellor_disputables import WAIT_PERIOD
 from tellor_disputables.alerts import alert
-from tellor_disputables.alerts import get_from_number
-from tellor_disputables.alerts import get_phone_numbers
+from tellor_disputables.alerts import generic_alert
+from tellor_disputables.alerts import get_twilio_info
+from tellor_disputables.data import chain_events
 from tellor_disputables.data import get_events
 from tellor_disputables.data import parse_new_report_event
 from tellor_disputables.utils import clear_console
-
+from tellor_disputables.utils import get_tx_explorer_url
+from tellor_disputables.utils import Topics
 
 warnings.simplefilter("ignore", UserWarning)
+logger = logging.getLogger("telliot_feeds.sources.price_aggregator")
+logger.handlers = [h for h in logger.handlers if not isinstance(h, logging.StreamHandler)]
+logging.basicConfig(filename="log.txt", level=logging.INFO, format="%(asctime)s %(message)s")
```

Comment:
should this be removed, or should it be `logger` instead of `logging`?

Reply:
- Added this cause the api calls were printing to the console instead of redirecting to log.txt.  Do you have that issue or was that on purpose?  Thought it would be cleaner if it didn't print to the console. 
- Was trying to avoid more imports if unnecessary and this seems to do the job.  Not sure if importing get_logger from telliot-feeds would make a difference.

Reply:
all good, does this close #78 ?